### PR TITLE
Add net-snmp project

### PR DESCRIPTION
**Pull request by fenner**
I started with the fuzzer snippet that you provided to the net-snmp project, expanded it with some experience (for debugging during replication), and added another packet parser fuzzer.

### CODE REVIEW

**Comment by fenner** [comment]
> Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

If this is true, I recommend updating https://github.com/google/oss-fuzz/blob/master/CONTRIBUTING to not say the opposite:

> You don't have to sign the CLA until after you've submitted your code for review and a member has approved it

In any case, I have signed the CLA.

File: projects/net-snmp/snmp_pdu_parse_fuzzer.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+    return 0;
+}
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+    unsigned char *data_ptr = (unsigned char *)calloc(1, size);
+    size_t bytes_remaining = size;
+    netsnmp_pdu *pdu = SNMP_MALLOC_TYPEDEF(netsnmp_pdu);
+
+    memcpy(data_ptr, data, size);
```
Comment (inferno-chromium) [inline projects/net-snmp/snmp_pdu_parse_fuzzer.c]: Same comment as above.

File: projects/net-snmp/agentx_parse_fuzzer.c [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    return 0;
+}
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+    unsigned char *data_ptr = (unsigned char *)calloc(1, size);
+    netsnmp_pdu *pdu = SNMP_MALLOC_TYPEDEF(netsnmp_pdu);
+    netsnmp_session session;
+
+    session.version = AGENTX_VERSION_1;
+    memcpy(data_ptr, data, size);
```
Comment (inferno-chromium) [inline projects/net-snmp/agentx_parse_fuzzer.c]: Why do you need to memcpy here, copying the same thing again just slows fuzzer execution. Please use reinterpret_cast.

Comment (fenner) [reply projects/net-snmp/agentx_parse_fuzzer.c]: I copied this from the code that Google supplied in a bug report to the net-snmp project. I'll happily remove it.

Comment (inferno-chromium) [reply projects/net-snmp/agentx_parse_fuzzer.c]: Should be an easy fix. Sometimes folks miss on optimizations when writing a fuzz target. We see this trend a lot, so recommending this for optimal performance.

Comment (fenner) [reply projects/net-snmp/agentx_parse_fuzzer.c]: Sure - don't leave bad examples around, since those will inevitably be the one someone picks to copy :-)

**Pull request merged**